Cartopy not building on Py3 windows

```
BUILD START: cartopy-0.13.0-np18py34_0
Fetching package metadata: ........ 
Solving package specifications: ....... 
The following packages will be downloaded:
 
    package                    |            build
    ---------------------------|-----------------
    geos-3.4.2                 |                3         4.6 MB  scitools
    proj4-4.9.1                |                0         499 KB  scitools
    cython-0.23.1              |           py34_0         1.9 MB  defaults
    numpy-1.9.2                |           py34_0        22.3 MB  defaults
    shapely-1.5.8              |       np19py34_2         680 KB  scitools
    pyepsg-0.2.0               |           py34_0           7 KB  scitools
    pyshp-1.2.1                |           py34_0          26 KB  scitools
    owslib-0.9.0               |           py34_0         197 KB  scitools
    ------------------------------------------------------------
                                           Total:        30.2 MB
 
The following NEW packages will be INSTALLED:
 
    cython:          0.23.1-py34_0    defaults
    geos:            3.4.2-3          scitools
    numpy:           1.9.2-py34_0     defaults
    owslib:          0.9.0-py34_0     scitools
    pip:             7.1.2-py34_0     defaults
    proj4:           4.9.1-0          scitools
    pyepsg:          0.2.0-py34_0     scitools
    pyshp:           1.2.1-py34_0     scitools
    python:          3.4.3-0          defaults
    python-dateutil: 2.4.2-py34_0     defaults
    pytz:            2015.4-py34_0    defaults
    requests:        2.7.0-py34_0     defaults
    setuptools:      18.1-py34_0      defaults
    shapely:         1.5.8-np19py34_2 scitools
    six:             1.9.0-py34_0     defaults
    wheel:           0.24.0-py34_0    defaults
 
Removing old work directory
Cloning into bare repository '/cygdrive/c/conda/conda-bld/git_cache/__github.com_SciTools_cartopy.git'...
checkout: 'v0.13.0'
Cloning into 'C:\conda\conda-bld\work'...
done.
Note: checking out 'v0.13.0'.
 
You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.
 
If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:
 
  git checkout -b new_branch_name
 
HEAD is now at bee9244... v0.13.0 version string
==> git log -n1 <==
 
commit bee9244abaaeb4480f87059446408533721e3275
Author: pelson <pelson.pub@gmail.com>
Date:   Wed Jul 1 15:49:34 2015 +0100
 
    v0.13.0 version string
 
 
==> git describe --tags --dirty <==
 
v0.13.0-dirty
 
 
==> git status <==
 
HEAD detached at v0.13.0
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)
 
	modified:   lib/cartopy/examples/always_circular_stereo.py
	modified:   lib/cartopy/examples/eyja_volcano.py
	modified:   lib/cartopy/examples/global_map.py
	modified:   lib/cartopy/examples/hurricane_katrina.py
	modified:   lib/cartopy/examples/rotated_pole.py
	modified:   lib/cartopy/examples/waves.py
	modified:   lib/cartopy/io/img_nest.py
	modified:   lib/cartopy/io/img_tiles.py
	modified:   tools/feature_download.py
 
no changes added to commit (use "git add" and/or "git commit -a")
 
 
Applying patch: 'C:\\projects\\conda-recipes-scitools-311\\cartopy\\cartopy.win.patch'
patching file `setup.py'
Package: cartopy-0.13.0-np18py34_0
source tree in: C:\conda\conda-bld\work
 
C:\conda\conda-bld\work>call "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\vcvarsall.bat" amd64 
The specified configuration type is missing.  The tools for the
configuration might not be installed.
 
C:\conda\conda-bld\work>REM ===== end generated header ===== 
 
C:\conda\conda-bld\work>set LIB=C:\conda\envs\_build\Library\lib;c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Lib\amd64;C:\Program Files\Microsoft SDKs\Windows\v7.1\Lib\X64; 
 
C:\conda\conda-bld\work>set LIBPATH=C:\conda\envs\_build\Library\lib;c:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Windows\Microsoft.NET\Framework\v4.0.30319;C:\Windows\Microsoft.NET\Framework64\v3.5;C:\Windows\Microsoft.NET\Framework\v3.5;;c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Lib\amd64; 
 
C:\conda\conda-bld\work>set INCLUDE=C:\conda\envs\_build\Library\include;c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE;C:\Program Files\Microsoft SDKs\Windows\v7.1\INCLUDE;C:\Program Files\Microsoft SDKs\Windows\v7.1\INCLUDE\gl; 
 
C:\conda\conda-bld\work>C:\conda\envs\_build\python.exe setup.py install 
setup.py:168: UserWarning: Unable to determine GEOS version. Ensure you have 3.3.3 or later installed, or installation may fail.
  '.'.join(str(v) for v in GEOS_MIN_VERSION), ))
running install
running bdist_egg
running egg_info
creating lib\Cartopy.egg-info
writing lib\Cartopy.egg-info\PKG-INFO
writing requirements to lib\Cartopy.egg-info\requires.txt
writing top-level names to lib\Cartopy.egg-info\top_level.txt
writing dependency_links to lib\Cartopy.egg-info\dependency_links.txt
writing manifest file 'lib\Cartopy.egg-info\SOURCES.txt'
reading manifest file 'lib\Cartopy.egg-info\SOURCES.txt'
writing manifest file 'lib\Cartopy.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build
creating build\lib.win-amd64-3.4
creating build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\crs.py -> build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\feature.py -> build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\img_transform.py -> build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\util.py -> build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\vector_transform.py -> build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\_epsg.py -> build\lib.win-amd64-3.4\cartopy
copying lib\cartopy\__init__.py -> build\lib.win-amd64-3.4\cartopy
creating build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\always_circular_stereo.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\arrows.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\barbs.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\eccentric_ellipse.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\effects_of_the_ellipse.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\eyja_volcano.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\favicon.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\features.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\feature_creation.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\geostationary.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\global_map.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\hurricane_katrina.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\image_tiles.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\logo.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\regridding_arrows.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\rotated_pole.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\srtm_shading.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\star_shaped_boundary.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\streamplot.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\tick_labels.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\tube_stations.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\un_flag.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\waves.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\wms.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\wmts.py -> build\lib.win-amd64-3.4\cartopy\examples
copying lib\cartopy\examples\__init__.py -> build\lib.win-amd64-3.4\cartopy\examples
creating build\lib.win-amd64-3.4\cartopy\io
copying lib\cartopy\io\img_nest.py -> build\lib.win-amd64-3.4\cartopy\io
copying lib\cartopy\io\img_tiles.py -> build\lib.win-amd64-3.4\cartopy\io
copying lib\cartopy\io\ogc_clients.py -> build\lib.win-amd64-3.4\cartopy\io
copying lib\cartopy\io\shapereader.py -> build\lib.win-amd64-3.4\cartopy\io
copying lib\cartopy\io\srtm.py -> build\lib.win-amd64-3.4\cartopy\io
copying lib\cartopy\io\__init__.py -> build\lib.win-amd64-3.4\cartopy\io
creating build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\clip_path.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\feature_artist.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\geoaxes.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\gridliner.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\patch.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\slippy_image_artist.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\ticker.py -> build\lib.win-amd64-3.4\cartopy\mpl
copying lib\cartopy\mpl\__init__.py -> build\lib.win-amd64-3.4\cartopy\mpl
creating build\lib.win-amd64-3.4\cartopy\sphinxext
copying lib\cartopy\sphinxext\gallery.py -> build\lib.win-amd64-3.4\cartopy\sphinxext
copying lib\cartopy\sphinxext\summarise_package.py -> build\lib.win-amd64-3.4\cartopy\sphinxext
copying lib\cartopy\sphinxext\__init__.py -> build\lib.win-amd64-3.4\cartopy\sphinxext
creating build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_coastline.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_coding_standards.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_crs.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_crs_transform_vectors.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_img_nest.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_img_tiles.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_img_transform.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_linear_ring.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_line_string.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_polygon.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_shapereader.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_util.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\test_vector_transform.py -> build\lib.win-amd64-3.4\cartopy\tests
copying lib\cartopy\tests\__init__.py -> build\lib.win-amd64-3.4\cartopy\tests
creating build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_albers_equal_area.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_azimuthal_equidistant.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_geostationary.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_lambert_conformal.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_mercator.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_robinson.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_rotated_geodetic.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_rotated_pole.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_stereographic.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\test_transverse_mercator.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
copying lib\cartopy\tests\crs\__init__.py -> build\lib.win-amd64-3.4\cartopy\tests\crs
creating build\lib.win-amd64-3.4\cartopy\tests\io
copying lib\cartopy\tests\io\test_downloaders.py -> build\lib.win-amd64-3.4\cartopy\tests\io
copying lib\cartopy\tests\io\test_ogc_clients.py -> build\lib.win-amd64-3.4\cartopy\tests\io
copying lib\cartopy\tests\io\test_srtm.py -> build\lib.win-amd64-3.4\cartopy\tests\io
copying lib\cartopy\tests\io\__init__.py -> build\lib.win-amd64-3.4\cartopy\tests\io
creating build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_axes.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_caching.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_crs.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_examples.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_features.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_gridliner.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_images.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_img_transform.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_mpl_integration.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_patch.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_pseudo_color.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_set_extent.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_shapely_to_mpl.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_ticker.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_ticks.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\test_web_services.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
copying lib\cartopy\tests\mpl\__init__.py -> build\lib.win-amd64-3.4\cartopy\tests\mpl
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_crs
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_crs\lambert_conformal_south.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_crs
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_crs\mercator_squashed.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_crs
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_examples
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_examples\global_map.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_examples
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_features
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_features\gshhs_coastlines.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_features
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_features\natural_earth.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_features
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_features\natural_earth_custom.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_features
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_features\wfs.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_features
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_gridliner
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_gridliner\gridliner1.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_gridliner
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_gridliner\gridliner_labels.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_gridliner
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_images
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_images\image_merge.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_images
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_images\image_nest.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_images
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_images\imshow_natural_earth_ortho.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_images
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_images\imshow_regional_projected.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_images
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_images\web_tiles.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_images
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_img_tiles2
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_img_tiles2\web_tiles.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_img_tiles2
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_img_transform
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_img_transform\regrid_image.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_img_transform
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\barbs_plate_carree.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\barbs_regrid.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\barbs_regrid_with_extent.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\global_contourf_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\global_contour_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\global_map.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\global_pcolor_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\global_scatter_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\multiple_projections1.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\natural_earth_interface.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\pcolormesh_global_wrap1.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\pcolormesh_global_wrap2.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\pcolormesh_global_wrap3.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\pcolormesh_goode_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\pcolormesh_limited_area_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\pcolormesh_mercator_wrap.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\quiver_plate_carree.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\quiver_regrid.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\quiver_regrid_with_extent.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\quiver_rotated_pole.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\simple_global.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration\streamplot.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_mpl_integration
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_shapely_to_mpl
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_shapely_to_mpl\contour_with_interiors.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_shapely_to_mpl
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_shapely_to_mpl\poly_interiors.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_shapely_to_mpl
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_ticks
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_ticks\xticks_cylindrical.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_ticks
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_ticks\xticks_no_transform.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_ticks
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_ticks\xyticks.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_ticks
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_ticks\yticks_cylindrical.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_ticks
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_ticks\yticks_no_transform.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_ticks
creating build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_web_services
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_web_services\wms.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_web_services
copying lib\cartopy\tests\mpl\baseline_images\mpl\test_web_services\wmts.png -> build\lib.win-amd64-3.4\cartopy\tests\mpl\baseline_images\mpl\test_web_services
creating build\lib.win-amd64-3.4\cartopy\data
creating build\lib.win-amd64-3.4\cartopy\data\raster
creating build\lib.win-amd64-3.4\cartopy\data\raster\natural_earth
copying lib\cartopy\data\raster\natural_earth\50-natural-earth-1-downsampled.png -> build\lib.win-amd64-3.4\cartopy\data\raster\natural_earth
creating build\lib.win-amd64-3.4\cartopy\data\raster\sample
copying lib\cartopy\data\raster\sample\Miriam.A2012270.2050.2km.jpg -> build\lib.win-amd64-3.4\cartopy\data\raster\sample
copying lib\cartopy\data\raster\sample\Miriam.A2012270.2050.2km.README.txt -> build\lib.win-amd64-3.4\cartopy\data\raster\sample
creating build\lib.win-amd64-3.4\cartopy\data\netcdf
copying lib\cartopy\data\netcdf\HadISST1_SST_update.nc -> build\lib.win-amd64-3.4\cartopy\data\netcdf
copying lib\cartopy\data\netcdf\HadISST1_SST_update.README.txt -> build\lib.win-amd64-3.4\cartopy\data\netcdf
creating build\lib.win-amd64-3.4\cartopy\data\shapefiles
creating build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs
copying lib\cartopy\data\shapefiles\gshhs\README.TXT -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs
creating build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\c
copying lib\cartopy\data\shapefiles\gshhs\c\GSHHS_c_L1.dbf -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\c
copying lib\cartopy\data\shapefiles\gshhs\c\GSHHS_c_L1.shp -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\c
copying lib\cartopy\data\shapefiles\gshhs\c\GSHHS_c_L1.shx -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\c
creating build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\l
copying lib\cartopy\data\shapefiles\gshhs\l\GSHHS_l_L2.dbf -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\l
copying lib\cartopy\data\shapefiles\gshhs\l\GSHHS_l_L2.shp -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\l
copying lib\cartopy\data\shapefiles\gshhs\l\GSHHS_l_L2.shx -> build\lib.win-amd64-3.4\cartopy\data\shapefiles\gshhs\l
copying lib\cartopy\io\srtm.json -> build\lib.win-amd64-3.4\cartopy\io
running build_ext
cythoning lib/cartopy/trace.pyx to lib/cartopy\trace.cpp
building 'cartopy.trace' extension
creating build\temp.win-amd64-3.4
creating build\temp.win-amd64-3.4\Release
creating build\temp.win-amd64-3.4\Release\lib
creating build\temp.win-amd64-3.4\Release\lib\cartopy
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -I. -I./lib/cartopy -IC:\conda\envs\_build\include -IC:\conda\envs\_build\include /Tplib/cartopy\trace.cpp /Fobuild\temp.win-amd64-3.4\Release\lib/cartopy\trace.obj
trace.cpp
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE\xlocale(323) : warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc 
lib/cartopy\trace.cpp(2324) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\trace.cpp(2342) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\trace.cpp(2360) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\trace.cpp(2534) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\trace.cpp(2552) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\trace.cpp(2570) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -I. -I./lib/cartopy -IC:\conda\envs\_build\include -IC:\conda\envs\_build\include /Tplib/cartopy/_trace.cpp /Fobuild\temp.win-amd64-3.4\Release\lib/cartopy/_trace.obj
_trace.cpp
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE\xlocale(323) : warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc 
lib/cartopy/_trace.cpp(308) : warning C4267: 'argument' : conversion from 'size_t' to 'unsigned int', possible loss of data 
lib/cartopy/_trace.cpp(325) : warning C4267: 'argument' : conversion from 'size_t' to 'unsigned int', possible loss of data 
lib/cartopy/_trace.cpp(454) : warning C4800: 'char' : forcing value to bool 'true' or 'false' (performance warning) 
lib/cartopy/_trace.cpp(456) : warning C4800: 'char' : forcing value to bool 'true' or 'false' (performance warning) 
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\link.exe /DLL /nologo /INCREMENTAL:NO /LIBPATH:. /LIBPATH:C:\conda\envs\_build\libs /LIBPATH:C:\conda\envs\_build\PCbuild\amd64 proj.lib geos_c.lib /EXPORT:PyInit_trace build\temp.win-amd64-3.4\Release\lib/cartopy\trace.obj build\temp.win-amd64-3.4\Release\lib/cartopy/_trace.obj /OUT:build\lib.win-amd64-3.4\cartopy\trace.pyd /IMPLIB:build\temp.win-amd64-3.4\Release\lib/cartopy\trace.lib /MANIFESTFILE:build\temp.win-amd64-3.4\Release\lib/cartopy\trace.pyd.manifest
trace.obj : warning LNK4197: export 'PyInit_trace' specified multiple times; using first specification
   Creating library build\temp.win-amd64-3.4\Release\lib/cartopy\trace.lib and object build\temp.win-amd64-3.4\Release\lib/cartopy\trace.exp
cythoning lib/cartopy/_crs.pyx to lib/cartopy\_crs.c
building 'cartopy._crs' extension
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -I. -IC:\conda\envs\_build\lib\site-packages\numpy\core\include -IC:\conda\envs\_build\include -IC:\conda\envs\_build\include /Tclib/cartopy\_crs.c /Fobuild\temp.win-amd64-3.4\Release\lib/cartopy\_crs.obj
_crs.c
c:\conda\envs\_build\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION 
lib/cartopy\_crs.c(5773) : warning C4244: 'function' : conversion from 'npy_intp' to 'long', possible loss of data 
lib/cartopy\_crs.c(14155) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\_crs.c(14173) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\_crs.c(14191) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\_crs.c(14631) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\_crs.c(14649) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
lib/cartopy\_crs.c(14667) : warning C4146: unary minus operator applied to unsigned type, result still unsigned 
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\link.exe /DLL /nologo /INCREMENTAL:NO /LIBPATH:. /LIBPATH:C:\conda\envs\_build\libs /LIBPATH:C:\conda\envs\_build\PCbuild\amd64 proj.lib /EXPORT:PyInit__crs build\temp.win-amd64-3.4\Release\lib/cartopy\_crs.obj /OUT:build\lib.win-amd64-3.4\cartopy\_crs.pyd /IMPLIB:build\temp.win-amd64-3.4\Release\lib/cartopy\_crs.lib /MANIFESTFILE:build\temp.win-amd64-3.4\Release\lib/cartopy\_crs.pyd.manifest
_crs.obj : warning LNK4197: export 'PyInit__crs' specified multiple times; using first specification
   Creating library build\temp.win-amd64-3.4\Release\lib/cartopy\_crs.lib and object build\temp.win-amd64-3.4\Release\lib/cartopy\_crs.exp
proj.lib(PJ_bipc.obj) : error LNK2019: unresolved external symbol __imp_hypot referenced in function s_forward
proj.lib(PJ_aeqd.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_geos.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(proj_etmerc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_eqdc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_bonne.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_aea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_lcc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_laea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_gnom.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_sterea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_ortho.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_oea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_mod_ster.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_sconics.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_stere.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_nsper.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_tpeqd.obj) : error LNK2001: unresolved external symbol __imp_hypot
build\lib.win-amd64-3.4\cartopy\_crs.pyd : fatal error LNK1120: 1 unresolved externals
error: command 'c:\\Program Files (x86)\\Microsoft Visual Studio 10.0\\VC\\Bin\\amd64\\link.exe' failed with exit status 1120
Command failed: C:\Windows\system32\cmd.exe /c call bld.bat
Command exited with code 1
```

--- a/proj4/meta.yaml
+++ b/proj4/meta.yaml
@@ -1,9 +1,18 @@
 package:
   name: proj4
   version: !!str 4.9.1
+
 source:
   fn:  proj-4.9.1.tar.gz
   url: http://download.osgeo.org/proj/proj-4.9.1.tar.gz
+
+build:
+  number: 1  # [win]
+
+requirements:
+  run:
+    # The VS version is important. See # https://github.com/SciTools/conda-recipes-scitools/issues/94.
+    - python  # [win]
 
 about:
   home: http://trac.osgeo.org/proj/

